pyaedt/modeler/circuits/object3dcircuit.py

```diff
@@ -871,6 +872,37 @@ def change_property(self, property_name, names=None):
             return self._oeditor.ChangeProperty(vOut)
         return False
 
+    @pyaedt_function_handler()
+    def enforce_touchstone_model_passive(self):
+        """Enforce touchstone model passive.
+
+        Returns
+        -------
+        bool
+
+        References
+        ----------
+
+        >>> oModelManager.EditWithComps
+        """
+        props = self.model_data.props
+        passive = OrderedDict(
+            [
+                ("DCOption", -1),
+                ("InterpOption", 1),
+                ("ExtrapOption", 3),
+                ("Convolution", 0),
+                ("Passivity", 6),
+                ("Reciprocal", False),
+                ("ModelOption", ""),
+                ("DataType", 2),
+            ]
+        )
+        props["NexximCustomization"] = passive
+        props["ModTime"] = int(time.time())
+        self.model_data.props = props
+        self.model_data.update()
+
```

Comment:
it should return True if the update() is correct,so do :

return self.model_data.update()